tests/unit/OptionsListUtilsTest.tsx

```diff
@@ -2828,4 +2827,92 @@ describe('OptionsListUtils', () => {
             expect(searchTerms2.includes(displayName)).toBe(true);
         });
     });
+
+    describe('DEW (Dynamic External Workflow)', () => {
```

Comment:
This is part of `getLastMessageTextForReport`, so let's move it inside the `describe('getLastMessageTextForReport', () => {`